### PR TITLE
PR#3

### PR DESCRIPTION
**Pull request by Galor-sw**

### CODE REVIEW

File: i18n.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  locales: ['en', 'he'],
+  defaultLocale: 'he',
+  pages: {
+    '*': ['common'],
+    '/': ['home'],
+    '/pre-form': ['pre-form'],
```
Comment (omrirz) [inline i18n.js:7]: What is this /pre-form? copied from another project? remove that.

File: components/Home-components/about-the-clinic.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            <img
+              src="/icons/about-clinic.png"
+              alt="אודות המרפאה"
+              className="justify-self-center"
+            />
+          </div>
+          <div className="grid place-items-center">
+            <div className="grid xl:place-items-start">
+              <div className="grid">
+                <a href="#" className="text-2xl text-header-blue xl:text-3xl">
```
Comment (omrirz) [inline components/Home-components/about-the-clinic.tsx:22]: in nextjs links should have <Link> wrapping the <a>.
https://nextjs.org/docs/api-reference/next/link

**Review by omrirz — CHANGES_REQUESTED** [review]
1. It has conflicts.
2. the font is not like in Figma.
3. when for submission succeed it should show a message below the button and not an alert().
4. phone input type should be tel.
5. The radius of a focused form input should be the same as the original radius
Screen Shot 2022-03-19 at 18 50 31
6. header spaces not equal
Screen Shot 2022-03-19 at 18 51 49
7. non equal spacing between sections (not just the attached image but all the sections)
Screen Shot 2022-03-19 at 18 53 58
8. the footer should be visible on mobile.
9. input fields should have inside padding.
Screen Shot 2022-03-19 at 18 56 17
10. look at spacing difference in Figma and implementation (fix it everywhere)
Screen Shot 2022-03-19 at 18 59 24
Screen Shot 2022-03-19 at 19 09 40
ng">

11. The colors should be those on the right in Figma (the blue and not the purple).
12. what is "jaso backup" file?
13. Never do this:
<ImgCircle className={`${otherSide}-0 -bottom-24`} />
Do:
<ImgCircle className={`${otherSide ? "left-0" : "right-0"} -bottom-24`} />
Why? because with ${otherSide}-0 tailwind will purge left-0 and right-0.
Also here: <ImgCircle className={`-${side}-36`} />
14. file names are still not consistent.
we have "Home-components" instead of "home-components", "Types.tsx" instead of "types.tsx"
15. What do we need next-translate for? If we're not using it then remove it.

**Comment by omrirz** [comment]
@Galor-sw Do we use next-translate? If not remove this i18n.js file.

File: i18n.js (hunk 1 above)
Comment (Galor-sw) [reply i18n.js]: removed

**Pull request closed without merging**